### PR TITLE
feat(logos): add black and white version for unbound

### PR DESCRIPTION
**Pull request by christiemolloy**
## Motivation
#397 

## What
Add black and white logos for the unbound service

## Why
Part of the new design update

## How
Add black and white svgs and add a new method to get the new svgs that is called inside of the unbound mobile services component.

## Verification Steps
 
1. Bind a new mobile service and you should see that the color of the icon changes/

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

Screen Shot 2019-09-03 at 11 25 03 AM

### CODE REVIEW

**Comment by tiffanynolan** [comment]
@christiemolloy Wanted to confirm. Is this just for the icon color change? If yes, those look great to me. 

I'm noticing some alignment issues with how the icons line up with the title/description though, especially with the Identity Management one.

**Comment by tiffanynolan** [comment]
@finp Related but not a blocker for this RE the descriptions being used in the mobile-services.js file. Do you have alternative, "better" descriptions for these services? I can make an issue to look into this further if you'd like. lmk.

**Comment by christiemolloy** [comment]
@tiffanynolan specifically just for the icon color changes, the alignment is handled in a separate branch!

File: server/mobile-services.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const PushService = {
   type: PUSH_SERVICE_TYPE,
   name: 'Push Notification',
   icon: '/img/push.svg',
+  iconBlackAndWhite: '/img/push-b&w.svg',
   description: 'Unified Push Server',
```
Comment (finp) [inline server/mobile-services.js]: @wei-lee  any thoughts on this?

**Comment by finp** [comment]
@tiffanynolan  I think it might require a separate issue re. language, eg can we get rid of the 'bind' verb now that it doesn't have any function in the mobile UX?

**Pull request merged**